Unit tests are failing on latest tag

Unit test outputs on travis have many instances of:
```
>           templated = template_main([project_filename], template=template_fp)
E           TypeError: main() got an unexpected keyword argument 'template'
```
sourced from https://github.com/pcdshub/pytmc/blob/master/pytmc/tests/test_commandline.py

It looks like we changed the `template` kwarg to `templates`: https://github.com/pcdshub/pytmc/blob/85fa86d9a716651aee70831e5bb8f69cc9566eed/pytmc/bin/template.py#L680
But I'm very confused because the master was passing with this in it

Failing tags:
https://app.travis-ci.com/github/pcdshub/pytmc/builds/254953822
https://app.travis-ci.com/github/pcdshub/pcds-envs/builds/255486106

Was passing on master prior to the tag (with failing docs...):
https://app.travis-ci.com/github/pcdshub/pytmc/builds/254263773

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -4,3 +4,4 @@ include README.rst
 include pytmc/templates/*
 include pytmc/_version.py
 include LICENSE
+recursive-include pytmc/tests *.txt

--- a/pytmc/tests/conftest.py
+++ b/pytmc/tests/conftest.py
@@ -15,6 +15,7 @@ TMC_FILES = list(TMC_ROOT.glob('*.tmc'))
 INVALID_TMC_FILES = list((TMC_ROOT / 'invalid').glob('*.tmc'))
 PROJ_ROOT = TEST_PATH / 'projects'
 TSPROJ_PROJECTS = list(str(fn) for fn in TEST_PATH.glob('**/*.tsproj'))
+TEMPLATES = TEST_PATH / 'templates'
 
 
 @pytest.fixture(scope='module')

--- a/pytmc/tests/templates/basic_test.txt
+++ b/pytmc/tests/templates/basic_test.txt
@@ -1,0 +1,1 @@
+{% for project in projects %}{{ project }}{% endfor %}

--- a/pytmc/tests/templates/smoke_test.txt
+++ b/pytmc/tests/templates/smoke_test.txt
@@ -1,0 +1,24 @@
+{% for project_path, project in projects.items() %}
+    {% set nc = get_nc(project) %}
+    {% for box in get_boxes(project) %}
+        box: {{ box.attributes["Id"] }}
+    {% endfor %}
+
+    {% for dt in get_data_types(project) %}
+        data type: {{ dt }}
+    {% endfor %}
+
+    {% for link in get_links(project) %}
+        link: {{ link.a }}
+    {% endfor %}
+
+    {% for plc in project.plcs %}
+        {% set results = get_linter_results(plc) %}
+        Pragma count: {{ results.pragma_count }}
+
+        {% for lib in get_library_versions(plc) %}
+            library: {{ lib }}
+        {% endfor %}
+
+    {% endfor %}
+{% endfor %}

--- a/pytmc/tests/test_commandline.py
+++ b/pytmc/tests/test_commandline.py
@@ -1,4 +1,3 @@
-import io
 import sys
 
 import pytest
@@ -86,8 +85,7 @@ def test_code(project_filename):
 
 def test_template_basic(project_filename):
     template = '{% for project in projects %}{{ project }}{% endfor %}'
-    with io.StringIO(template) as template_fp:
-        templated = template_main([project_filename], templates=template_fp)
+    templated = template_main([project_filename], templates=[template])
 
     print('templated', templated)
     assert templated == project_filename
@@ -127,7 +125,6 @@ def test_template_basic(project_filename):
     ],
 )
 def test_template_smoke(project_filename, template):
-    with io.StringIO(template) as template_fp:
-        templated = template_main([project_filename], templates=template_fp)
+    templated = template_main([project_filename], templates=[template])
 
     print('templated', templated)

--- a/pytmc/tests/test_commandline.py
+++ b/pytmc/tests/test_commandline.py
@@ -1,3 +1,4 @@
+import os
 import sys
 
 import pytest
@@ -12,6 +13,8 @@ from pytmc.bin.summary import main as summary_main
 from pytmc.bin.template import main as template_main
 from pytmc.bin.types import create_types_gui
 from pytmc.bin.xmltranslate import main as xmltranslate_main
+
+from .conftest import TEMPLATES
 
 
 def test_help_main(monkeypatch):
@@ -84,47 +87,28 @@ def test_code(project_filename):
 
 
 def test_template_basic(project_filename):
-    template = '{% for project in projects %}{{ project }}{% endfor %}'
-    templated = template_main([project_filename], templates=[template])
+    template = str(TEMPLATES / 'basic_test.txt')
+    templated = template_main(
+        [project_filename],
+        templates=[template + os.pathsep],
+    )
 
     print('templated', templated)
-    assert templated == project_filename
+    assert templated[template] == project_filename
 
 
 @pytest.mark.parametrize(
     'template',
     [
         pytest.param(
-            """
-            {% for project_path, project in projects.items() %}
-                {% set nc = get_nc(project) %}
-                {% for box in get_boxes(project) %}
-                    box: {{ box.attributes["Id"] }}
-                {% endfor %}
-
-                {% for dt in get_data_types(project) %}
-                    data type: {{ dt }}
-                {% endfor %}
-
-                {% for link in get_links(project) %}
-                    link: {{ link.a }}
-                {% endfor %}
-
-                {% for plc in project.plcs %}
-                    {% set results = get_linter_results(plc) %}
-                    Pragma count: {{ results.pragma_count }}
-
-                    {% for lib in get_library_versions(plc) %}
-                        library: {{ lib }}
-                    {% endfor %}
-
-                {% endfor %}
-            {% endfor %}
-            """,
+            str(TEMPLATES / 'smoke_test.txt'),
             id='helpers'),
     ],
 )
 def test_template_smoke(project_filename, template):
-    templated = template_main([project_filename], templates=[template])
+    templated = template_main(
+        [project_filename],
+        templates=[template + os.pathsep],
+    )
 
     print('templated', templated)

--- a/pytmc/tests/test_commandline.py
+++ b/pytmc/tests/test_commandline.py
@@ -87,7 +87,7 @@ def test_code(project_filename):
 def test_template_basic(project_filename):
     template = '{% for project in projects %}{{ project }}{% endfor %}'
     with io.StringIO(template) as template_fp:
-        templated = template_main([project_filename], template=template_fp)
+        templated = template_main([project_filename], templates=template_fp)
 
     print('templated', templated)
     assert templated == project_filename
@@ -128,6 +128,6 @@ def test_template_basic(project_filename):
 )
 def test_template_smoke(project_filename, template):
     with io.StringIO(template) as template_fp:
-        templated = template_main([project_filename], template=template_fp)
+        templated = template_main([project_filename], templates=template_fp)
 
     print('templated', templated)

